develop/parachain-devs/runtime-development/FRAME/frame-overview.md

```diff
@@ -0,0 +1,110 @@
+---
+title: FRAME Overview
+description: FRAME is a powerful toolkit for Polkadot SDK runtime development, offering modular pallets and support libraries. Learn how to build your runtime logic.
+---
+
+# FRAME Overview
+
+## Introduction
+
+The runtime in a Polkadot SDK-based blockchain acts as the core component, encapsulating all essential business logic for:
+
+- Executing transactions
+- Managing state transitions
+- Facilitating node interactions
+
+Polkadot SDK provides a comprehensive toolkit for constructing essential blockchain components, allowing developers to concentrate on crafting the specific runtime logic that defines their blockchain's unique behavior and capabilities.
+
+FRAME (Framework for Runtime Aggregation of Modularized Entities) is one of the most powerful tools available to you as a runtime developer. It consists of:
```

Comment:
I'd dive a bit deeper into what FRAME is just for more context. I was thinking of something like:

```suggestion
FRAME (Framework for Runtime Aggregation of Modularized Entities) provides a robust collection of tools to facilitate Polkadot SDK-based blockchain development. It offers reusable modules and useful abstractions that streamline developers' development process. It consists of:
```